geest/gui/panels/tree_panel.py

```diff
@@ -731,7 +798,16 @@ def _start_workflows(self, parent_item, role=None):
         """
         for i in range(parent_item.childCount()):
             child_item = parent_item.child(i)
-            self.queue_workflow_task(child_item, role)
+            # Skip items with "✔️" status if skip_completed is True
+            if role == child_item.role and (
+                not skip_completed or child_item.getStatus() != "✔️"
```

Comment:
I would not check for completion this way. Look for the presence of a result_file in the data(3) dict. We can write another helper to clear that before we start a job executing so that they do not  present as false positives.